src/bin/main.mts

```diff
@@ -39,12 +40,13 @@ program
     'package template',
     Defaults.PACKAGE_TEMPLATE,
   )
+  .option('--datadog', 'Configure application to be monitored by DataDog', Defaults.INCLUDE_DATADOG)
```

Comment:
Oh, looks like `-g` isn't actually an available option

Reply:
Oh right, yeah I must have thought the same thing and just removed that. Should we do `-dd, --datadog`?

Reply:
I think just `--datadog` is good.